src/TwigYard/Component/ConfigCacheServiceFactory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace TwigYard\Component;
+
+use Nette\Caching\Cache;
+use Nette\Caching\Storages\DevNullStorage;
+use Nette\Caching\Storages\FileStorage;
+
+class ConfigCacheServiceFactory
+{
+    /**
+     * @param LoggerFactory $loggerFactory
+     * @param $appRoot
+     * @param $cacheDir
+     * @param $cacheNamespace
+     * @param $cacheEnabled
+     * @return ConfigCache
+     */
+    public static function createConfigCache(
```

Comment:
Lets please add type hints to all newly added or modified methods. The task is to slowly start adding them and eventually have them everywhere.

Reply:
Added